src/lib.rs

```diff
@@ -247,9 +343,170 @@ fn simplify_imports(parsed_module: &mut Module) {
     }
 }
 
+#[derive(Serialize, Debug, Eq, PartialEq)]
+pub struct Range {
+    start: usize,
+    end: usize,
+}
+
+impl From<&Span> for Range {
+    fn from(value: &Span) -> Self {
+        Range {
+            start: value.lo.0 as usize - 1,
+            end: value.hi.0 as usize - 1,
```

Comment:
It appears that SWC spans are 1-based for some reason, while our existing API expects them to be 0-based. Or that there is a bug in the swc fork. Either way, this fixes the issue and satisfies the existing expectations. If it is indeed a bug upstream and we fix it, we can potentially remove the `-1` here but we shouldn't change the expectations in the tests.

Reply:
Yes, they are weirdly 1-based and they reserve 0 to mean the dummy span.